### DISCUSSION TITLE
Add PDF as a file type that can be handled

### DISCUSSION THREAD

**Discussion by joeprivettsmart** [remix-run/remix#4464] [Proposals]
Please could you add PDF as a type of file that can be handled by the build process and extracted to the __assets folder?

https://github.com/remix-run/remix/blob/main/packages/remix-dev/modules.ts

**Comment by machour** [comment]
👍🏼 

Would you like to open a pull request to the `dev` branch adding support for it?
Here's an example with the zip file extension: https://github.com/remix-run/remix/pull/3985
